test/obb/events/game_events_test.clj

```diff
@@ -21,29 +21,20 @@
   (let [system (-> (game-events-system) component/start)
```

Comment:
actually I feel the same that @quimrstorres talked about. usually I use the thread macro with more than 2 actions.. but feel free to ignore :)